cluster-sync/sync.sh

```diff
@@ -14,6 +14,7 @@ source ./cluster-sync/${KUBEVIRT_PROVIDER}/provider.sh
 CDI_INSTALL="install-operator"
 CDI_NAMESPACE=${CDI_NAMESPACE:-cdi}
 CDI_INSTALL_TIMEOUT=${CDI_INSTALL_TIMEOUT:-120}
+CDI_AVAILABLE_TIMEOUT=${CDI_AVAILABLE_TIMEOUT:-480}
```

Comment:
how can we make this longer for the crio lane?

Reply:
See https://github.com/kubevirt/project-infra/pull/401 we increase it just for the CRIO lane.